src/pathways/pathway/dto/create-pathway.dto.ts

```
@@ -89,4 +92,48 @@ export class CreatePathwayDto {
+  @ApiPropertyOptional({
+    description: "Whether a user can hold multiple active assignments of this pathway simultaneously. Must be true for VOLUNTEER type.",
+    example: false,
+    default: false,
+  })
+  @Expose()
+  @IsBoolean()
+  @IsOptional()
+  allow_multiple_active?: boolean;
+
+  @ApiPropertyOptional({
+    description: "Duration in months for which volunteer status is valid after completion. Required when type = VOLUNTEER.",
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)